src/WaCore.Crud.Contracts/Data/IWacListDataRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WaCore.Crud.Contracts.Data
+{
+    public interface IWacListDataRepository<TEntity, TFilter>
+    {
+        List<TEntity> GetAll(TFilter filter);
```

Comment:
It's better to have IList interface instead of simple List.

Reply:
I'm already changed this (including other changes), but I'm going to make a separate PR for that.